The nodeadkeys variant of Mac layouts is unusable, modifiers and others
important keys are not defined.

Submitted upstream: https://bugs.freedesktop.org/show_bug.cgi?id=8123

Index: xkb-data/rules/base.mlv_s.part
===================================================================
--- xkb-data.orig/rules/base.mlv_s.part
+++ xkb-data/rules/base.mlv_s.part
@@ -1 +1 @@
- $macs		$macvendorlayouts	nodeadkeys	=	macintosh_vndr/us(extended)+macintosh_vndr/%l%(v)
+ $macs		$macvendorlayouts	nodeadkeys	=	pc(pc105)+macintosh_vndr/us(extended)+macintosh_vndr/%l%(v)
